LICENSE.txt

```diff
@@ -530,6 +530,16 @@ Licensing information: © Autodesk, Inc. All Rights Reserved.
 
 The Artifakt font software is Autodesk proprietary and confidential, and may be used only by authorized users and only for Autodesk business purposes. Any use not authorized by Autodesk is not permitted and is an infringement of Autodesk's intellectual property rights as well as a breach of your agreement with Autodesk. Go to https://brand.autodesk.com/brand-system/typography for detailed usage guidelines on when and how to use the Artifakt designer collection.
 
+Lib.Harmony v.2.2.1
+Copyright (c) 2017 Andreas Pardeike
```

Comment:
can you add the MIT license type here as well so it's easy to identify without comparing legalese?